src/index.d.ts

```diff
@@ -0,0 +1,2248 @@
+// Type definitions for react-native-elements 0.16
```

Comment:
@iRoachie is this something that will have to be maintained and kept up to date with each new version? 

What's the process of generating a new typed `index.d.ts` file?

Reply:
Yes! It's not generated automatically. Has to be maintained manually. I don't mind being in charge  for that

Reply:
Great. That works for me. If it gets too cumbersome, esp with the changes on v1.. I don't mind helping out. lmk.